client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-how-to-migrate/diy-option/style.scss

```diff
@@ -1,5 +1,12 @@
-.how-to-migrate__experiment-diy {
+.how-to-migrate__experiment-diy:not(.step-container-v2 *) {
 	color: var(--studio-gray-90);
 	font-weight: 500;
 	text-decoration: underline;
 }
+
+.site-migration {
+	.step-container-v2__heading {
+		max-width: 620px;
+		margin: 0 auto;
+	}
+}
```

Comment:
This CSS seems to blog to the flow, not the step. We should put a new `style.scss` file next to the flow and add this CSS.

Reply:
Currently, we'll apply those changes to only two flows indeed, but that should change soon as we'll probably release this to all flows. Should we keep this connected to the step in this case?

@zaguiini, can you confirm that step container v2 will be released to all flows?

Reply:
Thanks for reviewing this @alshakero!

Currently, we'll apply those changes to only two flows indeed, but that should change soon as we'll probably release this to all flows. Should we keep this connected to the step in this case?

@zaguiini, can you confirm that step container v2 will be released to all flows?

Reply:
My pleasure! I would just avoid adding flow names into steps. Even if we release it to every flow. Steps are literred with flow logic.

Reply:
(Copying my response shared in the designs file)

Most headings across pages are 6-cols width (616px), but in domain transfer step it could be bigger as the domain length.

I'm fine setting all headings to be 10-cols width, but we need to keep the line breaks as in the mockups so paragraphs don't exceed characters per line.

I'm also curious whether translations exist and if it's possible to see the outcome in languages that differ significantly (ie, German). My main concern there is breaking the line in H1 elements.

Reply:
Here are screenshots of the current UI in Dutch and German:

**Dutch**
![Screenshot 2025-03-26 at 14 04 50](https://github.com/user-attachments/assets/84049e8c-fc6f-4adb-b244-25667e83c9a4)

**German**
![Screenshot 2025-03-26 at 14 05 40](https://github.com/user-attachments/assets/be66189b-c630-4967-9b7c-d4a3ede13e80)